json4s/src/main/scala/org/http4s/json4s/Json4sInstances.scala

```diff
@@ -10,10 +10,10 @@ import scala.util.control.NonFatal
 import scalaz.{EitherT, \/}
 
 trait Json4sInstances[J] {
-  implicit lazy val json: EntityDecoder[JValue] = jawn.jawnDecoder(facade)
+  implicit lazy val jsonDecoder: EntityDecoder[JValue] = jawn.jawnDecoder(facade)
```

Comment:
Not this PR's problem, but why is this `EntityDecoder` `lazy`? If its not important that it be lazy I'd prefer to avoid the overhead of a `lazy val`.